cmd/kube-apiserver/app/server.go

```diff
@@ -311,6 +311,17 @@ func CreateKubeAPIServerConfig(
 		return
 	}
 
+	// defaults to empty range and ip
+	var secondaryServiceIPRange net.IPNet
+	var secondaryApiServerServiceIP net.IP
+	// process secondary range only if provided by user
+	if s.SecondaryServiceClusterIPRange.IP != nil {
```

Comment:
No feature gate check here?

Reply:
At this point, configuration has been validated at earlier stage leading to this code path.